common/djangoapps/microsite_configuration/admin.py

```diff
@@ -15,4 +15,24 @@ class Meta(object):  # pylint: disable=missing-docstring
         model = Microsite
 
 
+class MicrositeHistoryAdmin(admin.ModelAdmin):
```

Comment:
We can use the [config_models for this](https://github.com/edx/edx-platform/blob/master/common/djangoapps/config_models/models.py). And we get also a cache for free